src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int MAX_VALUE = 5;
```

Comment:
Consider renaming 'MAX_VALUE' to a more descriptive name such as 'FIGURE_COUNT' to reflect its purpose in the context of figures.